README.md

```diff
@@ -1077,6 +1078,26 @@ public Foo CurrentFoo
 }
 ```
 
+## [L.13] ❌ **DO NOT** Use LINQ query syntax
```

Comment:
Do we need any explanation for why?

Reply:
I was thinking about that and I do think it could have an explanation for consistency/readability. In the end though, I think it's a preference thing because there's no performance hit.

Reply:
I'm happy to add an explanation. Do you have a preference of what it should say?

Reply:
Not entirely, but I do think it should talk about consistency withing the file and overall readability. I think we should loop @gregr-ni and @ccifra into this discussion

Reply:
To me it is about consistency. We don't need multiple ways to do the same thing and that eliminates another syntax that people have to look out for when reading code.

Reply:
I agree with Greg's response here.

Reply:
How's this for an explanation?
>Do not use query/SQL syntax for LINQ expressions; use method/object syntax instead.
>This is to promote consistency within our code.

Reply:
I would say something along the lines of "Having only one way to read/write LINQ expressions increases consistency in our code"